eclipse-theia/theia
# Stub proposed APIs used by Python debugger

#### What it does

Closes https://github.com/eclipse-theia/theia/issues/13881

Stubs the APIs listed in the [package.json](https://github.com/microsoft/vscode-python-debugger/blob/444cbb37b3c75fcdadf60787a079779571df7d19/package.json#L7-L11) of the vscode-python-debugger plugin.

#### How to test

1. Download and install the `Python` extension suite (that includes the `Python Debug` extension)
2. Open a Python file.
3. There should be no notification that the extension crashes during startup.
4. You could also test that debugging works as expected for a simple python file.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
